packages/devtools_app/lib/src/screens/performance/panes/queued_microtasks/queued_microtasks_view.dart

```diff
@@ -85,60 +67,52 @@ class RefreshQueuedMicrotasksInstructions extends StatelessWidget {
   }
 }
 
-// In the response returned by the VM Service, microtasks are sorted in
-// ascending order of when they will be dequeued, i.e. the microtask that will
-// run earliest is at index 0 of the returned list. We use those indices of the
-// returned list to sort the entries of the microtask selector, so that they
-// they also appear in ascending order of when they will be dequeued.
-typedef IndexedMicrotask = (int, Microtask);
+/// In the response returned by the VM Service, microtasks are sorted in
+/// ascending order of when they will be dequeued, i.e. the microtask that will
+/// run earliest is at index 0 of the returned list. We use those indices of the
```

Comment:
nit: blank line after first sentence of dart doc

Reply:
Done.